test/distributed/fsdp/test_utils.py

```diff
@@ -169,11 +169,11 @@ def test_module_wrap_policy(self):
         """
         model = self.Model(TestGetSubmoduleToStates.SharedParameterMode.PARENT_CHILD)
 
```

Comment:
These changes are just renaming from `submodule` to `fully_sharded_module` (missed from the previous PR introducing "fully sharded module").